source/enketo.md

```diff
@@ -39,3 +39,5 @@ iOS (running on iPhones and iPads) devices have several known limitations with u
 * Version 8.x shows "undefined is not an object (evaluating 'c.resources')". Solution: Update to latest iOS version
 
 If data collection offline is not required and you are seeing an error on iOS, consider using the [online-only version](https://ee.kobotoolbox.org/::ABCD) instead of the offline URL. 
```

Comment:
I know this isn't part of your current change, but I see that this link doesn't lead to anything useful 👍 Maybe just fix this up in the same PR and then I will merge.